### ISSUE TITLE
ValueError: Internal error - config has not been correctly loaded. Over 3.2.1

### ISSUE THREAD

**Issue by lyz-code** [smarie/python-pytest-cases#196]
Since pytest-cases 3.2.1 I'm getting the following error:

```
----------------
- Testing code -
----------------
pytest --cov-report term-missing --cov src tests 
ImportError while loading conftest '/tmp/tmp.PxdGvyGRrZ/repository-orm/tests/conftest.py'.
tests/conftest.py:78: in <module>
    ) -> Tuple[Any, Repository, Repository, RepositoryTester]:
../env/lib/python3.7/site-packages/pytest_cases/common_pytest.py:843: in apply
    return apply_decorator(test_or_fixture_func, container)
../env/lib/python3.7/site-packages/pytest_cases/case_parametrizer_new.py:142: in _apply_parametrization
    argvalues = get_parametrize_args(host_class_or_module, cases_funs, prefix=prefix, debug=debug, scope=scope)
../env/lib/python3.7/site-packages/pytest_cases/case_parametrizer_new.py:308: in get_parametrize_args
    return [c for _f in cases_funs for c in case_to_argvalues(host_class_or_module, _f, prefix, scope, debug)]
../env/lib/python3.7/site-packages/pytest_cases/case_parametrizer_new.py:308: in <listcomp>
    return [c for _f in cases_funs for c in case_to_argvalues(host_class_or_module, _f, prefix, scope, debug)]
../env/lib/python3.7/site-packages/pytest_cases/case_parametrizer_new.py:335: in case_to_argvalues
    meta = MiniMetafunc(case_fun)
../env/lib/python3.7/site-packages/pytest_cases/common_pytest.py:595: in __init__
    raise ValueError("Internal error - config has not been correctly loaded. Please report")
E   ValueError: Internal error - config has not been correctly loaded. Please report
make: *** [Makefile:66: test-code] Error 4
```

# Steps to reproduce

```bash
git clone https://github.com/lyz-code/repository-orm
cd repository-orm
make install
make test # pytest-cases 3.1.1 -> Works
pip install pytest-cases==3.3.0
make test # Error is shown
```

I've tried debugging it without success. Can you please help me?

Thanks!

**Comment by smarie** [comment]
Thanks @lyz-code ! This seems to be caused by usage of cases in the `conftest.py`.

I found out that `conftest.py` files are loaded before the `pytest_configure` hook. I'll fix this now by simplyreading the early config in pytest_load_initial_conftests hook instead.

**Comment by smarie** [comment]
Fixed in 3.4.4. Let me know if this works for you ! Thanks again for letting me know

**Comment by lyz-code** [comment]
Yes it did! thanks for the quick fix :)
